build.gradle

```diff
@@ -9,6 +9,7 @@ repositories {
 dependencies {
     implementation 'junit:junit:4.13.1'
     testImplementation('org.junit.jupiter:junit-jupiter:5.6.0')
+    testImplementation 'org.testng:testng:7.1.0'
```

Comment:
I'm wondering what did this line of code change? Was it required in the testing?

Reply:
It kind of just added it and refuse to work with out it so I'm not sure why it did that